app/screens/send-bitcoin-screen/payment-destination/lnurl.ts

```diff
@@ -31,62 +31,56 @@ export const resolveLnurlDestination = async ({
   userDefaultWalletIdQuery,
   myWalletIds,
 }: ResolveLnurlDestinationParams): Promise<ParseDestinationResult> => {
-  if (parsedLnurlDestination.valid) {
-    try {
-      const lnurlParams = await getParams(parsedLnurlDestination.lnurl)
-      if ("reason" in lnurlParams) {
-        throw lnurlParams.reason
-      }
+  // TODO: Move all logic to galoy client or out of galoy client, currently lnurl pay is handled by galoy client
```

Comment:
I think all in client eventually make most sense. could be after we have done all the tests/iteration for it and we want to work on refactoring the web wallet

Reply:
Yeah I think that makes sense